Sources/Storage/Disk/Extension/String+Base64.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  String+MD5.swift
+//  StatsdClient
+//
+//  Created by Nghia Tran on 10/8/17.
+//  Copyright © 2017 StatsdClient. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+
+    func fromBase64() -> String? {
```

Comment:
I think the extension methods should be named `base64Encoded` and `base64Decoded`. Reasoning behind that is you always want to namespace your method with some prefix.

Reply:
Also public extensions should not be nested under `Disk` group

Reply:
Agree 👍